src/utils/notifications/requestNotification.js

```diff
@@ -59,6 +72,23 @@ class Notifications {
       }
     });
   }
+
+  /**
+   * @return {function} create notifications
+   */
+  async requestUpdated() {
+    await emitter.on('request update', async (request) => {
+      let notificationMessage;
+      let ownerId;
+      const { userId } = await this.makeNotification(request);
+      if (request.status !== 'Pending') {
+        notificationMessage = `Your Request has been ${request.status.toLowerCase()}`;
+        ownerId = userId;
+      }
+
+      this.notify(notificationMessage, ownerId, 'updated', request.id);
+    });
+  }
```

Comment:
I think we should move the emailing part of the feature to this function and check if the user hasn't opted out of emails. Or add the check to the controller

Reply:
that has to be in the controller cause it was part of this story